persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/multinode/AbstractClusteredPersistentEntitySpec.scala

```diff
@@ -147,7 +147,8 @@ abstract class AbstractClusteredPersistentEntitySpec(config: AbstractClusteredPe
   }
   "A PersistentEntity in a Cluster" must {
 
-    "send commands to target entity" in within(10.seconds) {
+    "send commands to target entity" in within(11.seconds) {
+      enterBarrier("before-1")
```

Comment:
This commit undoes my previous commit (`try`+ `catch`) providing what I think is the correct contention strategy to one of the issues in this test class.

When the first test (`send commands to target entity`)  failed, we usually saw all other tests fail. That brought some confusion and proved there was some dependency between the tests. Adding a barrier at the beginning of each test instead of adding it at the end we achieve the following: 

 - if the current test succeeds on my node, then report it, start the next test but await there
 - if the current test fails on my node, I will probably exit the test sooner than expected so if there was a barrier at the end of the test I won't reach it.

Because there's an issue in `expectAppendCount` (I still need to investigate that) `JVM-1` was always missing the `after-1` barrier. That caused all sorts of side effects where other nodes would time out or unsynchronise or ... causing that either all tests passed or all tests failed.